cluster-operator/src/main/resources/log4j2.properties

```diff
@@ -1,21 +0,0 @@
-name = COConfig
- 
-appender.console.type = Console
-appender.console.name = STDOUT
-appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
-
-rootLogger.level = ${env:STRIMZI_LOG_LEVEL:-INFO}
-rootLogger.appenderRefs = stdout
-rootLogger.appenderRef.console.ref = STDOUT
-rootLogger.additivity = false
-
-# Kafka AdminClient logging is a bit noisy at INFO level
-logger.kafka.name = org.apache.kafka
-logger.kafka.level = ${env:STRIMZI_AC_LOG_LEVEL:-WARN}
-logger.kafka.additivity = false
-
-# Zookeeper is very verbose even on INFO level -> We set it to WARN by default
-logger.zookeepertrustmanager.name = org.apache.zookeeper
-logger.zookeepertrustmanager.level = ${env:STRIMZI_ZOOKEEPER_LOG_LEVEL:-WARN}
-logger.zookeepertrustmanager.additivity = false
```

Comment:
Should we still have some defaults when the config map is for whatever reason not mounted?